client/src/Routes.jsx

```diff
@@ -22,7 +23,7 @@ export default class Routes extends Component {
             component={Home}
           />
         </Route>
-
+        <Route path='/newsoftheday' component={NewsOfTheDays} />
```

Comment:
are u sure we need route? It is part of a home page

Reply:
removed